packages/ncurses.rb

```diff
@@ -63,11 +63,9 @@ def self.build
   def self.install
     Dir.chdir 'ncursesw_build' do
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-      FileUtils.ln_sf "#{CREW_LIB_PREFIX}/libtinfow.so.6", "#{CREW_DEST_LIB_PREFIX}/libtinfow.so.5"
     end
     Dir.chdir 'ncurses_build' do
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-      FileUtils.ln_sf "#{CREW_LIB_PREFIX}/libtinfo.so.6", "#{CREW_DEST_LIB_PREFIX}/libtinfo.so.5"
```

Comment:
We'll need new binaries to really get rid of the symlinks.  Please remove so new ones can be added.

Reply:
We'll also need to bump the version.

Reply:
> We'll need new binaries to really get rid of the symlinks. Please remove so new ones can be added.
Can you clarify what you mean by "we'll need new binaries"

> We'll also need to bump the version.
Oops. Thought I had done that.

Reply:
Turns out this isn't needed for this PR at all. Removed it and will create a separate PR later.